docs/user/interactive-running-locally.rst

```diff
@@ -0,0 +1,121 @@
+.. _local_interactive_environments:
+
+Running RenkuLab Interactive Sessions on Your Own Machine
+=========================================================
+
+Each Renku project contains configuration files (``Dockerfile``,
+``requirements.txt`` etc.) for building reproducible session images. We build
+these automatically when you push your project to the server and store them in
+the image registry connected to the Renku instance that you are using. The
+custom-built images are used whenever you launch an interactive session on
+RenkuLab.
+
+You can of course also use these same images to run your code in the correct
+environment anywhere that `Docker <https://www.docker.com>`_ is available.
+
+.. note::
+
+    All of the instructions below assume that you have cloned your project
```

Comment:
I would add `jupyter` to the local requirements

Reply:
everything runs in docker, so no need to have jupyter installed locally

Reply:
ops, you are right :sweat_smile: